builder

```diff
@@ -17,7 +17,7 @@ main() {
 			--libexecdir="$prefix/lib" \
 			--enable-multi-arch \
 			--enable-stack-protector=strong
-		make && make install
+		make -j$(nproc-1) && make install
```

Comment:
🙋 Why have you made this change?

Reply:
It just wants to speed up the building process since it is a little bit slow when using qemu to build.

Reply:
Does it matter?

Reply:
I think it results in building test failed in the previous version since the parallel build needs a lot of memory. But, I check the log, there is not enough memory. So, in latest version, I remove it.